server/filters/get-days.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 import moment from 'moment';
 
-export default function formatDate(date: Date): string {
+export default function getDays(date: Date): string {
```

Comment:
Yes that is correct. Most of the time on promos we dont display a time stamp. You only see that within the article. Unless the articles is featured in a list or transporter where the module title (eg Latest) references time, then we show time under the title.